### ISSUE TITLE
The 'flowcraft==1.3.0' distribution was not found and is required by the application

### ISSUE THREAD

**Issue by sjackman** [assemblerflow/flowcraft#134]
Commit https://github.com/assemblerflow/flowcraft/commit/2173530b52a64dcb7aeb9f2458835137d0bfb33a `Bumped version` was added after 1.3.0 was tagged.

Downloading and running the tagged `flowcraft` 1.3.1 tarball gives the error

```
pkg_resources.DistributionNotFound: The 'flowcraft==1.3.0' distribution was not found and is required by the application
```

Please bump the version to 1.3.2 and tag a new stable release.

**Comment by sjackman** [comment]
Never mind. I jumped to the wrong conclusion. flowcraft 1.3.1 installs fine. The only problem is that it reports the wrong version number.
```
❯❯❯ flowcraft --version
1.3.0
```

**Comment by ODiogoSilva** [comment]
Fixed by changing the 1.3.1 tag commit to https://github.com/assemblerflow/flowcraft/commit/2173530b52a64dcb7aeb9f2458835137d0bfb33a
